stream_alert_cli_module/runner.py

```diff
@@ -604,6 +596,8 @@ def configure_output(options):
     props = output.get_user_defined_properties()
 
     for name, prop in props.iteritems():
+        # TODO: Don't use protected method here
+        # pylint: disable=protected-access
```

Comment:
We could create a new copy of the namedtuple here, but I would be 100% okay with adding _replace to the ignored protected warnings.